docs/source/data-types

```diff
@@ -0,0 +1 @@
+../../book/src/data-types
```

Comment:
@lauranovich Since you already changed where the original docs are located in the GitHub Action (https://github.com/scylladb/scylla-rust-driver/blob/f3f02288b2137b72460afd5cff9cb8c3bf334db0/.github/workflows/book.yml), you can move all the files from ``book/src`` to ``docs/source`` without using redirects.

The book should still build, and you won't have to create new redirects every time there are new documentation files created.